docs/src/webview2.md

```diff
@@ -0,0 +1,128 @@
+---
+id: webview2
+title: "WebView2"
+---
+
+The following will explain how to use Playwright with [Microsoft Edge WebView2](https://docs.microsoft.com/en-us/microsoft-edge/webview2/). WebView2 is a WinForms control, which will use Microsoft Edge under the hood to render web content. It is a part of the Microsoft Edge browser and is available on Windows 10 and Windows 11. Playwright can be used to automate WebView2 applications and can be used to test web content in WebView2. For connecting to WebView2, Playwright uses [`method: BrowserType.connectOverCDP`] which connects to it via the Chrome DevTools Protocol (CDP).
+
+## Overview
+
+A WebView2 control can be instructed to listen on CDP by setting either the `WEBVIEW2_ADDITIONAL_BROWSER_ARGUMENTS` environment variable with `--remote-debugging-port=9222` or call [EnsureCoreWebView2Async](https://docs.microsoft.com/en-us/dotnet/api/microsoft.web.webview2.wpf.webview2.ensurecorewebview2async?view=webview2-dotnet-1.0.1343.22) with the `--remote-debugging-port=9222` argument. This will start the WebView2 process with the Chrome DevTools Protocol enabled.
+
+```txt
```

Comment:
csharp ?

Reply:
The issue if we add csharp, that the generator will filter it out for the other languages, since it thinks its only relevant for our dotnet language binding.

Reply:
Let's teach it not to if there is an extra keyword (generic?)?